### PR TITLE
[zh] Translate 'Security Chaos Engineering' into Chinese

### PR DESCRIPTION
**Pull request by devsachinsharma**
Tried my best.Any changes please let me know.

### CODE REVIEW

**Comment by jihoon-seo** [comment]
@devsachinsharma I have a question: did you use machine translator to translate this?
I am asking you because: it would be great for us to adopt localization jobs that are authored and reviewed by people who are fluent with that language.

File: content/zh-cn/security_chaos_engineering.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: 安全混沌工程
+status: Completed
+category: 概念
+---
+
+## 这是什么
+
+安全混沌工程或中骏是一门基于以下学科的学科：[Chaos Engineering](/chaos_engineering/) SCE 在分布式系统上执行主动安全实验建立对系统抵御湍流和恶意条件的能力的信心。安全混沌工程师使用科学的方法循环来实现这一点，包括稳态、假设、持续验证、经验教训和缓解实施。
```
Comment (hanyuancheung) [inline content/zh-cn/security_chaos_engineering.md:9]:
```suggestion
安全混沌工程 或 其简称 SCE 是一门基于[混沌工程](/zh-cn/chaos_engineering/)的学科。SCE 在分布式系统上执行主动安全实验建立对系统抵御湍流和恶意条件的能力的信心。安全混沌工程师使用科学的方法循环来实现这一点，包括稳态、假设、持续验证、经验教训和缓解实施。

```

File: content/zh-cn/security_chaos_engineering.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+category: 概念
+---
+
+## 这是什么
+
+安全混沌工程或中骏是一门基于以下学科的学科：[Chaos Engineering](/chaos_engineering/) SCE 在分布式系统上执行主动安全实验建立对系统抵御湍流和恶意条件的能力的信心。安全混沌工程师使用科学的方法循环来实现这一点，包括稳态、假设、持续验证、经验教训和缓解实施。
+
+## 它解决的问题
+
+主要优先事项 [site reliability engineers](/site-reliability-engineering/)（断续器）和网络安全工程师是尽快恢复服务，目标是实现零停机时间并最大限度地减少业务影响。SRE 和网络安全工程师同时处理故障前和故障后事件的情况。大多数安全问题都难以快速发现和修补，从而影响应用程序或系统功能。此外，在开发阶段，安全事件通常很难发现。
```
Comment (hanyuancheung) [inline content/zh-cn/security_chaos_engineering.md:13]:
```suggestion
主要优先事项 [site reliability engineers](/zh-cn/site-reliability-engineering/)（断续器）和网络安全工程师是尽快恢复服务，目标是实现零停机时间并最大限度地减少业务影响。SRE 和网络安全工程师同时处理故障前和故障后事件的情况。大多数安全问题都难以快速发现和修补，从而影响应用程序或系统功能。此外，在开发阶段，安全事件通常很难发现。

```

File: content/zh-cn/security_chaos_engineering.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+## 它如何帮助
+
+安全混沌工程是围绕 [observability](/observability/) 和网络弹性实践。它旨在发现“未知的未知”并建立对系统的信心，提高网络弹性并提高可观察性。
```
Comment (hanyuancheung) [inline content/zh-cn/security_chaos_engineering.md:17]:
```suggestion
安全混沌工程是围绕 [observability](/zh-cn/observability/) 和网络弹性实践。它旨在发现“未知的未知”并建立对系统的信心，提高网络弹性并提高可观察性。

```

File: content/zh-cn/security_chaos_engineering.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+工程团队将逐步提高对安全问题的理解在复杂的基础架构、平台和分布式系统中。中骏 提高了整个产品的网络弹性，发现了隐藏的安全问题，揭示经典盲点，并为关键边缘情况准备团队。这种方法有助于 断续器 ，[DevOps](/devops/) 和 [DevSecOps](/devsecops/) 工程师建立对系统的信心，提高网络弹性并提高可观察性。
```
Comment (hanyuancheung) [inline content/zh-cn/security_chaos_engineering.md:19]:
```suggestion
工程团队将逐步提高对安全问题的理解在复杂的基础架构、平台和分布式系统中。中骏 提高了整个产品的网络弹性，发现了隐藏的安全问题，揭示经典盲点，并为关键边缘情况准备团队。这种方法有助于 断续器 ，[DevOps](/zh-cn/devops/) 和 [DevSecOps](/zh-cn/devsecops/) 工程师建立对系统的信心，提高网络弹性并提高可观察性。

```

File: content/zh-cn/security_chaos_engineering.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+
+## 这是什么
+
+安全混沌工程或中骏是一门基于以下学科的学科：[Chaos Engineering](/chaos_engineering/) SCE 在分布式系统上执行主动安全实验建立对系统抵御湍流和恶意条件的能力的信心。安全混沌工程师使用科学的方法循环来实现这一点，包括稳态、假设、持续验证、经验教训和缓解实施。
+
+## 它解决的问题
+
+主要优先事项 [site reliability engineers](/site-reliability-engineering/)（断续器）和网络安全工程师是尽快恢复服务，目标是实现零停机时间并最大限度地减少业务影响。SRE 和网络安全工程师同时处理故障前和故障后事件的情况。大多数安全问题都难以快速发现和修补，从而影响应用程序或系统功能。此外，在开发阶段，安全事件通常很难发现。
+
+## 它如何帮助
```
Comment (hanyuancheung) [inline content/zh-cn/security_chaos_engineering.md:15]:
```suggestion
## 如何帮助

```

File: content/zh-cn/security_chaos_engineering.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+title: 安全混沌工程
+status: Completed
+category: 概念
+---
+
+## 这是什么
+
+安全混沌工程或中骏是一门基于以下学科的学科：[Chaos Engineering](/chaos_engineering/) SCE 在分布式系统上执行主动安全实验建立对系统抵御湍流和恶意条件的能力的信心。安全混沌工程师使用科学的方法循环来实现这一点，包括稳态、假设、持续验证、经验教训和缓解实施。
+
+## 它解决的问题
```
Comment (hanyuancheung) [inline content/zh-cn/security_chaos_engineering.md:11]:
```suggestion
## 解决的问题

```

File: content/zh-cn/security_chaos_engineering.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
+---
+title: 安全混沌工程
+status: Completed
+category: 概念
+---
+
+## 这是什么
```
Comment (hanyuancheung) [inline content/zh-cn/security_chaos_engineering.md:7]:
```suggestion
## 是什么

```

**Review by hanyuancheung — CHANGES_REQUESTED** [review]
Needs to review carefully and not use a translator or something like that.

**Comment by hanyuancheung** [comment]
Also, `.hugo_build.lock` file should not be committed. Pls git rm it.

**Comment by jihoon-seo** [comment]
Note: the filename should be modified to `content/zh-cn/security-chaos-engineering.md`.

File: content/zh-cn/security_chaos_engineering.md (hunk 4 above)
Comment (Submarinee) [reply content/zh-cn/security_chaos_engineering.md]: 工程团队将逐步提高对安全问题的理解在复杂的基础架构、平台和分布式系统中。中骏 提高了整个产品的网络弹性，发现了隐藏的安全问题，揭示经典盲点，并为关键边缘情况准备团队。这种方法有助于 断续器 ，[DevOps](/zh-cn/devops/) 和 [DevSecOps](/zh-cn/devsecops/) 工程师建立对系统的信心，提高网络弹性并提高可观察性。

> 
We'd better replace this word「中骏」  to SCE.

**Comment by Superskyyy** [comment]
I do not think this translate even after reviewer modifications would suffice the fluent requirement, I suggest it to be redone entirely.

**Comment by hanyuancheung** [comment]
According to advice from others, this PR is closing cause of too much uncomfortable work. Pls redo it or anyone interested is welcome to contribute.

**Pull request closed without merging**